openprescribing/media/js/test/test_measures.js

```diff
@@ -0,0 +1,743 @@
+            centiles, options, 6);
+          expect(result[0].chart_title).to.equal('ACE');
+          expect(result[0].chart_title_url).to.equal('/ccg/03V/ace');
+          str = '<strong>Cost savings:</strong> If it had prescribed in line ';
+          str += 'with the median, this CCG would have spent £10.00 less ';
+          str += 'over the past 6 months.';
+          expect(result[0].chart_explanation).to.equal(str);
+        });
+
+        it('sets the expected title, URL, and descriptions for measure practice charts', function () {
+          var data = [
+            {
+              id: 'ace',
+              name: 'ACE',
+              mean_percentile: 80,
+              cost_saving_50th: 10,
+              is_cost_based: true,
+              data: []
+            }
+          ],
+          globalData = [{ id: 'ace', data: []}],
+          globalCentiles = [],
+          centiles = ['10'],
+          options = {
+              rollUpBy: 'measure_id',
+              orgType: 'practice',
+              orgId: 'A81001',
+              parentOrg: '03V'
+          };
+          var result = mu.addChartAttributes(data, globalData, globalCentiles,
+            centiles, options, 6);
+          expect(result[0].chart_title).to.equal('ACE');
+          expect(result[0].chart_title_url).to.equal('/ccg/03V/ace');
+          str = '<strong>Cost savings:</strong> If it had prescribed in line ';
+          str += 'with the median, this practice would have spent £10.00 less ';
+          str += 'over the past 6 months.';
+          expect(result[0].chart_explanation).to.equal(str);
+        });
+    });
+
+    describe('#_addHighchartsXAndY', function () {
+        it('should convert global data correctly', function () {
+          var data = [
+            { date: '2015-01-01', percentiles: { 'ccg': { '10': 2.1} }},
+            { date: '2015-02-01', percentiles: { 'ccg': { '10': 2.4} }},
+            { date: '2015-03-01', percentiles: { 'ccg': { '10': 2.2} }}
+          ];
+          var options = {'orgType': 'ccg'};
+          var result = mu._addHighchartsXAndY(data, true, false, options, '10');
+          expect(result.length).to.equal(3);
+          expect(result[0].x).to.equal(1420070400000);
+          expect(result[0].y).to.equal(2.1);
+        });
+
+        it('should multiply percentages by 100 for display purposes', function () {
+          var data = [
+            { date: '2015-01-01', calc_value: '0.08'},
+            { date: '2015-02-01', calc_value: '0.04'},
+            { date: '2015-03-01', calc_value: '0.05'}
+          ];
+          var options = {};
+          var result = mu._addHighchartsXAndY(data, false, true, options, null);
+          expect(result.length).to.equal(3);
+          expect(result[0].x).to.equal(1420070400000);
+          expect(result[0].y).to.equal(8);
+        });
+    });
+
+    describe('#_getChartTitleEtc', function () {
+        it('should get explanation for non-cost-based measures', function () {
```

Comment:
I don't understand this test (or its description): isn't this a cost-based measure?